.github/workflows/merge.yml

```diff
@@ -0,0 +1,53 @@
+---
+
+name: Merge
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy_flatpak:
+    name: Deploy Flatpak
+    runs-on: ubuntu-latest
+
+    container:
+      image: docker.io/bilelmoussaoui/flatpak-github-actions
+      options: --privileged
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build
+        uses: bilelmoussaoui/flatpak-github-actions/flatpak-builder@flat-manager
+        with:
+          bundle: calculator.flatpak
+          manifest-path: io.elementary.calculator.yml
+          run-tests: true
+          repository-name: elementary
+          repository-url: https://flatpak.elementary.io/elementary.flatpakrepo
+          cache-key: flatpak
+
+      - name: Deploy
```

Comment:
I'm not sure why this is not done on the CI stage but only triggered if it's a merge event against the master branch?

Reply:
Yes, I just didn't see why the need to duplicate the build steps here instead of just having one file for flatpak that does build & deploy.

Reply:
It's easier to have each file for an event instead of having a file for what you want to do and filter by `if`s.

- CI - runs on PR and PR code change
- Merge - runs on merging to master
- Release - runs on a new release created

Moving deploy to CI means CI will need to run on `push`, which means it gets ran twice for every PR in the org (don't hurt, but is also very pointless), or we have some very weird `if` conditions. Not to mention, we would still have a `Merge` file to handle gettext stuff.